src/Element/TraversableElement.php

```diff
@@ -157,7 +157,11 @@ public function fillField($locator, $value)
             throw new ElementNotFoundException($this->getDriver(), 'form field', 'id|name|label|value|placeholder', $locator);
         }
 
-        $field->setValue($value);
+        if ($field->getTagName() === "select") {
```

Comment:
Actually it's DomCrawler who is doing all the setting: https://github.com/symfony/dom-crawler/tree/master/Field